internal-client/src/main/java/org/apache/uniffle/client/request/RssAccessClusterRequest.java

```diff
@@ -17,18 +17,34 @@
 
 package org.apache.uniffle.client.request;
 
+import java.util.Collections;
+import java.util.Map;
 import java.util.Set;
 
 public class RssAccessClusterRequest {
 
   private final String accessId;
   private final Set<String> tags;
   private final int timeoutMs;
+  /**
+   * The map is to pass the extra data to the coordinator and to
+   * extend more pluggable {@code AccessCheckers} easily.
+   */
+  private final Map<String, String> extraProperties;
 
   public RssAccessClusterRequest(String accessId, Set<String> tags, int timeoutMs) {
     this.accessId = accessId;
     this.tags = tags;
     this.timeoutMs = timeoutMs;
+    this.extraProperties = Collections.emptyMap();
+  }
+
+  public RssAccessClusterRequest(String accessId, Set<String> tags, int timeoutMs,
+          Map<String, String> extraProperties) {
```

Comment:
Either all parameters on the next line or one new line per parameter.

Reply:
done